index.html

```diff
@@ -47,14 +47,15 @@ <h2 class="pt-5">Przygotuj się na zmiany</h2>
          <img src="./logo.png" alt="">
        </div>
        <div class="col-md-9">
-         <h1 class="pb-5">Jak jest na P1</h1>
```

Comment:
@LukaszekW - żeby zwiększyć odstępy możesz używać klas z bootstrapa: https://getbootstrap.com/docs/4.4/utilities/spacing/ do zwiększania marginów i paddingów (zamiast `<br/>`). Będzie to wtedy zachowywać się prawidłowo w każdej rozdzielczości (mobile i desktop).